tracing-subscriber/src/fmt/format/pretty.rs

```diff
@@ -174,13 +174,12 @@ where
                 if let Some(name) = thread.name() {
                     write!(writer, "{}", name)?;
                     if self.display_thread_id {
-                        write!(writer, " ({:?})", thread.id())?;
+                        write!(writer, " ")?;
```

Comment:
minor nit: prefer `writer.write_char()` over `write!` when writing just a single character:
```suggestion
                        writer.write_char(' ')?;
```